package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "@demos-europe/demosplan-ui",
-  "version": "0.0.2",
+  "name": "my-webpack-project",
+  "version": "1.0.0",
   "license": "MIT",
   "private": false,
-  "description": "Vue components, Vue directives, Design Token and Scss files to build interfaces for demosPlan.",
+  "description": "My webpack project",
```

Comment:
"description": "Vue components, Vue directives, Design Token and Scss files to build interfaces for demosPlan.",

Reply:
this and the following concerns in relation to package.json have been adjusted here [e72971f](https://github.com/demos-europe/demosplan-ui/pull/13/commits/e72971fc72bf1d65b82123a9ad58a4ab23b167e2)